koku_setup_on_m1.rst

```diff
@@ -0,0 +1,135 @@
+=====================
+Koku Setup on M1 Mac
+=====================
+
+About
+=====
+
+This documentation is to guide M1 Mac users on how to successfully install and run Koku on their machines.
+
+Getting Started
+================
+
+This guide mostly makes use of Rosetta in order to get Intel-based programs and packages to run on Mac with Apple silicon processors (M1).
+
+Using Rosetta (2)
+-----------------
+
+Rosetta 2 enables a Mac with Apple silicon to use apps built for a Mac with an Intel processer. Check out and follow the steps on `how to install Rosetta on your Mac`_.
+
+TL;DR: ::
+
+    /usr/sbin/softwareupdate --install-rosetta --aggree-to-license
+
+1. Running Terminal with Rosetta
+--------------------------------
+
+It is a good idea to have a separate terminal to run with Rosetta, while keeping the default terminal run on M1.
+
+1. Download and install `iTerm2`_
+2. Go to Applications in Finder
+3. Find iTerm, right-click and select `Get Info`
+4. Make sure `Open using Rosetta` is checked
+
+Alternatively, you can create a copy of your default terminal, rename the copy and have it run specifically with Rosetta.
+
+`Note`: We will be using this version of the terminal for all the steps that follow.
+
+1. Installing ``brew``
+----------------------
+
+Same as we have a separate terminal running Intel-based programs, we also want to have a separate ``brew`` version that run Intel-based programs. When you install Homebrew on an Intel Mac, it installs it in the `/usr/local/homebrew` directory.
+
+1. Create a ``~/Downloads/homebrew`` and download Homebrew tarball and extract it to the ``~/Downloads/homebrew`` directory: ::
+
+    cd ~/Downloads
+    mkdir homebrew
+    curl -L https://github.com/Homebrew/brew/tarball/master | tar xz --strip 1 -C homebrew
+
+2. Move the ``homebrew`` directory to ``/usr/local/homebrew``. You need the ``sudo`` command: ::
+
+    sudo mv homebrew /usr/local/homebrew
+
+3. Add the following path and alias to the ``.zshrc`` file: ::
```

Comment:
We might want to add a note here that we are aliasing brew to brow so that no one is confused in the following steps when we are using `brow` as the command.